python/example_code/iam/iam_basics/README.md

```diff
@@ -1,8 +1,37 @@
-# AWS Identity and Access Management (IAM) examples
+# AWS Identity and Access Management (IAM) getting started examples
```

Comment:
Can use short name in title, then spell out on first mention. 
**IAM** getting started examples